assets/model_monitoring/components/src/shared_utilities/event_utils.py

```diff
@@ -5,12 +5,18 @@
 
 
 import logging
-import mlflow
-from azureml._restclient.models import BaseEvent
-from azureml._restclient.run_client import RunClient
-from azureml._restclient.service_context import ServiceContext
-from azureml.core.authentication import AzureMLTokenAuthentication
-from mlflow import MlflowClient
+import warnings
+try:
+    import mlflow
+    from azureml._restclient.models import BaseEvent
+    from azureml._restclient.run_client import RunClient
+    from azureml._restclient.service_context import ServiceContext
+    from azureml.core.authentication import AzureMLTokenAuthentication
+    from mlflow import MlflowClient
```

Comment:
Are these required for the metrics calculation?

Reply:
this isn't in the GSQ component, it's in some shared utilities to post messages for dpv2 jobs

Reply:
ok I removed the imports I think it might work now that I've changed the GSQ CI to test specific files instead of the entire directory